traffic_portal/test/integration/PageObjects/UsersPage.po.ts

```diff
@@ -46,13 +68,39 @@ export class UsersPage extends BasePage {
     private txtPassword = element(by.name('uPass'));
     private txtConfirmPassword = element(by.name('confirmPassword'));
     private txtPublicSSHKey = element(by.name('publicSshKey'));
+    private txtSearch = element(by.id('usersTable_filter')).element(by.css('label input'));
+    private btnTableColumn = element(by.css('[title="Select Table Columns"]'));
     private randomize = randomize;
 
-    async OpenUserPage(){
-      let snp = new SideNavigationPage();
-      await snp.ClickUserAdminMenu();
-      await snp.NavigateToUsersPage();
-     }
+    async OpenUserPage() {
+        let snp = new SideNavigationPage();
+        await snp.NavigateToUsersPage();
+    }
+
+    async OpenUserMenu() {
+        let snp = new SideNavigationPage();
+        await snp.ClickUserAdminMenu();
+    }
+
+    public async CheckCSV(name: string): Promise<boolean> {
+        return element(by.cssContainingText("span", name)).isPresent();
+    }
+
+    public async CheckToggle(name: string): Promise<boolean> {
+        let result = false;
+        await this.btnTableColumn.click();
+        //if the box is already checked, uncheck it and return false
+        if (await browser.isElementPresent(element(by.xpath("//th[text()='" + name + "']"))) === true) {
```

Comment:
nit: instead of using an XPath here, you could just do like you did above with the CheckCSV span:

```typescript
if (await element(by.cssContainingText("th", name)).isPresent()) {
    // ...
}
```